### DISCUSSION TITLE
Multi-language examples

### DISCUSSION THREAD

**Discussion by queengooborg** [mdn/mdn-community#321] [Platform]
It would be awesome if Yari allowed for multi-language examples; that is, the same examples in multiple coding languages.  Google's API documentation, GitHub Docs, Selenium.dev and more provide examples in multiple languages with tabs above to switch between them.

A good example of a page we currently have that would benefit from this is https://developer.mozilla.org/en-US/docs/Web/WebDriver.  In this page, we include a Python example for setting up a Selenium WebDriver test, but Selenium offers libraries for NodeJS, Java and more, so it would be amazing to include examples in those languages as well.

**Comment by Rumyra** [comment]
cc @fiji-flo
